### ISSUE TITLE
Duplicate Ad notifications are not shown in the notification drawer

### ISSUE THREAD

**Issue by tmancey** [brave/browser-android-tabs#1615]
## Steps to reproduce 

   1. Enable Ads
   2. Visit www.brave.com
   3. Wait for duplicate Ad notifications to appear (see Additional Information on how to reduce the wait time for an Ad to appear)

## Actual result 
Duplicate Ad notifications are not shown in the notification drawer

## Expected result
Duplicate Ad notifications should be shown in the notification drawer

## Issue reproduces how often 
Easily reproduced

## Issue happens on 
   - Current Play Store version? No
   - Beta build? No

## Device details
   - Install type (ARM, x86): ARM
   - Device (Phone, Tablet, Phablet): Pixel3 Phone
   - Android version: 9
   
## Brave version
Brave Ads

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? 
- Is the issue reproducible on the latest version of Chrome? 

### Additional information

You can reduce the time it takes for an Ad notification to appear by temporarily changing:
```
uint64_t start_timer_in =
    base::Time::kSecondsPerHour / ads_client_->GetAdsPerHour();
```

to

```
uint64_t start_timer_in = 30;  // Ad notification will be shown every 30 seconds
```

in `AdsImpl::StartDeliveringNotifications`

**Comment by tmancey** [comment]
This will be resolved as part of #1616

**Comment by GeetaSarvadnya** [comment]
- Verification passed on Samsung Galaxy J3 version 5.1.1 running BraveModernarm `v1.1.0-brave-ads-6`
![Screenshot_2019-07-05-19-53-05](https://user-images.githubusercontent.com/38657976/60673562-a65ded80-9e95-11e9-8684-ef7e4869d225.png)
